edifact-edi/2.2/modules/ROOT/edits/index.adoc

```diff
@@ -0,0 +1,107 @@
+
+== Prerequisites
+
+To use this connector, you must be familiar with:
+
+* Anypoint Connectors
+* Mule runtime engine (Mule)
+* Elements and global elements in a Mule flow
+* Creating a Mule app using Anypoint Studio (Studio)
+
+To use EDIFACT EDI Connector, you must have:
+
+* Purchased a MuleSoft license for Anypoint B2B
+
+* Anypoint Studio 7.0 or later
+
+[NOTE]
+
+EDIFACT Connector supports files of up to 15 MB in size, and the memory requirement is approximately 40 to 1.
+For example, a 1-MB file requires up to 40 MB of memory to process, so it's important to consider this memory requirement in conjunction with your Transactions Per Second (TPS)  needs for large X12 files.
+This is not an exact figure; the value varies according to the complexity of the mapping instructions.
+
+=== Understanding the EDI Schema Language
+
+The EDIFACT EDI Connector uses a YAML format called EDI Schema Language (ESL) to represent EDI schemas. Basic ESLs define the structure of EDI messages in terms of
+
+* structures, known as messages, in EDIFACT terminology
+
+* groups, segments, composites, and elements
+
+You can customize the EDIFACT schema definitions to suit your data by defining an overlay schema. An overlay schema is a type of ESL that enables you to modify a base schema, such as an EDIFACT D.98B ORDERS schema, with your specific conventions. You don't need an overlay schema if you're using the structure defined by the standard, but most EDI exchanges modify the base definitions in an implementation convention.
+
+You can also define your own schemas from scratch, or copy and edit a base EDIFACT schema definition to suit your data as shown in the  xref:x12-edi-connector::x12-edi-schema-language-reference.adoc[EDI Schema Language Reference].
+
+== Finding Schema Locations
+
+To use the connector, you need to know the locations of the schemas
+in your project. If you're using out-of-the-box EDIFACT schemas and
+not customizing anything, the schema location follows the
+`/edifact/{version}/{message}.esl` convention.
+For example, using the D.98B version of the ORDERS message, your schema location is `/edifact/d98b/ORDERS.esl`.
+
+If you're using one or more custom schemas, place these in the
+`src/main/resources` directory and refer to the location relative to this directory.
+For example, if you've put your ORDERS schema (either overlay or full) in `src/main/resources/mypartner/ORDERS.esl`,
+your schema location is `/mypartner/ORDERS.esl`.
+
+== The Hierarchy of EDIFACT Connector Message Structure
+
+Use this connector to read or write EDIFACT documents into (or from) the canonical EDI message structure. This structure is a hierarchy of Java Maps and Lists. You use either DataWeave or code to manipulate them. Each transaction has its own structure as defined in the schemas.
+
+The message contains the following keys, some of which apply to either the Read operation or Write operation, as indicated:
```

Comment:
```suggestion
The message contains the following keys, some of which apply to either the Read operation or the Write operation, as indicated:
```

Reply:
See above, and in the following table Description column, consistently capitalize Read operation and Write operation.